styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+    padding: 0 10px;
+  }
+  
+  header {
+    position: sticky;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  @media (max-width: 768px) {
+    header {
+      flex-direction: column;
+    }
+  }
+  
+  svg {
+    width: 300px;
+  }
+  
+  ul {
+    display: flex;
+    list-style: none;
+  }
+  
+  form {
+    text-align: center;
+  }
+  
+  a {
+    color: black;
+    text-decoration: none;
+    margin-right: 10px;
+  }
+  input {
+    padding: 10px;
+    border-radius: 15px
+  }
+  
+  section {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    padding-top: 56.25%; /* 16:9 Aspect Ratio (divide 9 by 16 = 0.5625) */
```

Comment:
Please do not use unused codes with comments. it makes your code very messy. Please try to make your code neat as possible.